packages/medusa/src/utils/authenticate-middleware.ts

```diff
@@ -82,8 +82,9 @@ export const authenticate = (
       authUser &&
       (isRegistered || (!isRegistered && options.allowUnregistered))
     ) {
-      req.auth_user = {
-        id: authUser.id,
+      req.auth = {
+        actor_id: getActorId(authUser, authScope) as string,
```

Comment:
NIT: Instead of casting as string here, you can set the return type of `getActorId` as a string